tensilelite/Tensile/Tests/common/gemm/use_sgpr_for_gro_with_dtva.yaml

```diff
@@ -0,0 +1,1130 @@
+TestParameters:
+  marks: [skip-gfx900, skip-gfx906, skip-gfx908, skip-gfx90a, skip-gfx1010, skip-gfx1011, skip-gfx1012, skip-gfx1030, skip-gfx1100, skip-gfx1101, skip-gfx1102, skip-gfx1200, skip-gfx1201] # not supported by arch
+
+GlobalParameters:
```

Comment:
don't create another testing yaml.
You should directly add the test into dtv

Reply:
fixed